guides/v2.3/inventory/source-selection-algorithms.md

```diff
@@ -0,0 +1,167 @@
+---
+group: inventory
+title: Source selection algorithms
+---
+
+The **Source Selection Algorithm (SSA)** receives the total number of shippable SKUs and their quantities in an order as well as the order's shipping address. It recommends how to fulfill partial and full shipments. The merchant decides which business needs take precedence when deciding which shipping method to use:
+
+* Should the products be delivered from the sources designated as having the highest priority? 
+* Should the total shipment cost be the primary factor in choosing a shipment method?
+* Should the shipments originate from the closest source?
```

Comment:
you could also mention that delivery time could be one of the decisive factors, I know that some merchants interested in the fastest way of delivery as an option